Build/Skins/illustro/Google/Google.ini

```diff
@@ -34,7 +34,7 @@ colorText=255,255,255,205
 ; However, it's not the easiest way to start learning Rainmeter. Get a
 ; hold of the basics, and then check out the manual entry for InputText.
 Measure=Plugin
-Plugin=InputText.dll
+Plugin=InputText

```

Comment:
The comment above should also have `.dll` removed.

Reply:
Thank you for the review, as always.  
I have fixed that.

Reply:
Doesn't seem to be applied. I was talking about this:
https://github.com/rainmeter/rainmeter/blob/55b75a812f531aaf7510c66aef732a785a3e1302/Build/Skins/illustro/Google/Google.ini#L33

Reply:
Done. Thanks!